packages/payments-client/src/components/pages/payment.tsx

```diff
@@ -51,12 +59,12 @@ export const PaymentsPage: React.FC = () => {
       {merchantKey && (
         <Formik
           initialValues={{
-            customerFirstName: '',
-            customerLastName: '',
-            address1: '',
-            city: '',
-            postalCode: '',
-            country: '',
+            customerFirstName: forename,
+            customerLastName: surname,
+            address1: buildingName || buildingNumber + line1,
```

Comment:
So it's my fault but when I gave you this line, was pseudo code. What I meant was to concatenate buildingNumber and line1 if they exist with a space and if not, just an empty string.

Logic is; if buildingName, and line 1 `${buildingName} ${line1}` else if bulidlingNumber and line1  `${buildingName} ${line1}` else if just buildingName or buildingNumber or just line1, render what you have